pkg/templates/ignition.go

```diff
@@ -98,6 +101,9 @@ func (i *ignition) GenerateNode(kluster *kubernikusv1.Kluster, pool *models.Node
 		for _, userLabel := range pool.Labels {
 			nodeLabels = append(nodeLabels, userLabel)
 		}
+		for _, userAnnotation := range pool.Annotations {
+			nodeAnnotations = append(nodeAnnotations, userAnnotation)
```

Comment:
I don't understand where this is used. This is a new variable and it I'm missing how this is passed to ignition.